docs/how-to/how-to-add-custom-secret.md

```diff
@@ -1,23 +1,30 @@
 # Add custom secret to Prow
 
-This tutorial shows how to add a custom secret and use it in the Prow pipeline.
+This tutorial shows how to add and use a custom secret in the Prow pipeline.
 ## Procedure
 
-1. Add a secret to the Google Secret Manager service and apply the necessary permissions. Add the `secret-manager-trusted@sap-kyma-prow.iam.gserviceaccount.com` principal with the `Secret Manager Secret Accessor` role if the secret will be used only for a postsubmit or release job. If you are creating a Secret for a presubmit job, use `secret-manager-untrusted@sap-kyma-prow.iam.gserviceaccount.com` principal with the same role. If you want to use the secret in presubmit and postsubmit jobs, apply both principals.
+1. Create a new secret in the Secret Manager. The secret name should be in the following format: `<component>-<tool>-<secret-name>`. For example: `security-dashboard-github-bot-token`. The secret should contain the following fields:
+   - **secret value** - the value of the secret, for example, the GitHub token.
+   - **description** - the description of the secret in the **Annotations description** field. For example: `GitHub token for the Security dashboard Github bot`.
+
+2. Apply the `owner` label to the secret in Secret Manager to help identify the secret owner. For example: `owner: neighbors`.
+3. Apply the `type` label to the secret in Secret Manager to help identify the secret owner. For example: `type: service-account`.
```

Comment:
A `type` label is not to identify the secret owner. Additionally a `service-account` is not a type of a secret. More suitable type would be service-account-key or even service-account jwt key.